pycbc/transforms.py

```diff
@@ -592,51 +596,50 @@ def inverse_transform(self, maps):
             of transformed values.
         """
         out = {}
-        out[parameters.chirp_distance] = \
-                conversions.chirp_distance(maps[parameters.distance],
-                                            maps[parameters.mchirp], ref_mass=self.ref_mass)
+        out[parameters.chirp_distance] = conversions.chirp_distance(
+            maps[parameters.distance], maps[parameters.mchirp], ref_mass=self.ref_mass
+        )
         return self.format_output(maps, out)
 
     def jacobian(self, maps):
         """Returns the Jacobian for transforming chirp distance to
         luminosity distance, given the chirp mass.
         """
-        ref_mass=1.4
-        mchirp = maps['mchirp']
-        return (2.**(-1./5) * self.ref_mass / mchirp)**(-5./6)
+        ref_mass = 1.4
+        mchirp = maps["mchirp"]
+        return (2.0 ** (-1.0 / 5) * self.ref_mass / mchirp) ** (-5.0 / 6)
 
     def inverse_jacobian(self, maps):
         """Returns the Jacobian for transforming luminosity distance to
         chirp distance, given the chirp mass.
         """
-        ref_mass=1.4
-        mchirp = maps['mchirp']
-        return (2.**(-1./5) * self.ref_mass / mchirp)**(5./6)
-    
-######################################################################################################### TEST ################################################################################################################################################
-import numpy as np
-from pycbc.waveform.waveform_modes import jframe_to_l0frame
+        ref_mass = 1.4
+        mchirp = maps["mchirp"]
+        return (2.0 ** (-1.0 / 5) * self.ref_mass / mchirp) ** (5.0 / 6)
+
+
+class AlignTotalSpin(BaseTransform):
 
-class alignTotalSpin(BaseTransform):
-    
     """Parameters
     ----------
-    iota =
-    
-    s1 =
-    
-    s2 =
-    
-    m1 =
-    
-    m2 =
-    
-    f_lower =
-    
-    phi_ref =
+    iota : str
+        The name of the iota parameter
+    s1 : str
+        The name of the s1 parameter
+    s2 : str
+        The name of the s2 parameter
+    m1 : str
+        The name of the m1 parameter
+    m2 : str
+        The name of the m2 parameter
+    f_lower : str
+        The name of the f_lower parameter
+    phi_ref : str
+        The name of the phi_ref parameter
     """
-    name = "Align_total_spin"
-    
+
+    name = "align_total_spin"
+
     def __init__(self, iota, s1, s2, m1, m2, f_lower, phi_ref, newIota, news1, news2):
```

Comment:
I think we should be passing in & out the x/y/z components of the spin, rather than packaging them into vectors.